pages/common/complete.md

```diff
@@ -0,0 +1,15 @@
+# complete
+
+> Provides argument autocompletion to shell commands.
+
+- Apply a function, that performs autocompletion, to a command:
```

Comment:
Don't think the commas after `function` and `command` are needed.

Reply:
Not a problem. I'll drop those.